src/Service/Users.php

```diff
@@ -30,11 +30,11 @@ class Users {
   protected $eventDispatcher;
 
   /**
-   * An instance of Config Factory.
+   * An instance of openam API operations.
    *
-   * @var \Drupal\Core\Config\Config|\Drupal\Core\Config\ImmutableConfig
+   * @var \Drupal\openam_api\Service\Users
    */
-  private $config;
+  private $openamApiOperations;
```

Comment:
Shouldnt this be `$openamConfig` instead of `$openamApiOperations`?
Given you have renamed 
```
$this->config = $configFactory->get('openam_api_operations.settings');
```
to 
```
$openamConfig = $configFactory->get('openam_api.settings');
```

Reply:
Updated as discussed.